user-guide/Reference/DataMiner_Tools/Data_Aggregator_Migrators.md

```diff
@@ -1,17 +1,59 @@
 ---
-uid: Data_Aggregator_Migrator
+uid: Data_Aggregator_Migrators
 ---
 
-# Data Aggregator Migrator
+# Data Aggregator Migrators
 
-The Data Aggregator Migrator should be used **after** upgrading the Data Aggregator DxM from version **2.x.x to 3.x.x** or higher. This **one-time migration process** is only necessary if the prior installation included [configured GQI queries](xref:Data_Aggregator_queries).
+Following migrators are available to make it easier to update existing setups to newer versions:
 
-> You can download the tool from [DataMiner Dojo](https://community.dataminer.services/download/data-aggregator-migrator/).
+* [Enabling GQI DxM](#enabling-gqi-dxm)
+* [Upgrading from version 2.x.x](#upgrading-from-version-2xx)
 
 > [!TIP]
 > See also: [Data Aggregator](xref:Data_Aggregator_DxM)
 
-## Using the Data Aggregator Migrator
+## Enabling GQI DxM
+
+The Data Aggregator MigratorToGQIDxM tool should be used to migrate any existing queries into the format supported by the GQI DxM.
+
+1. Make sure Data Aggregator version 3.1.0 or higher is installed.
+
+1. Enable the [`UseGQIDxM`](xref:Data_Aggregator_settings#gqi-dxm) setting.
+
+1. Restart the `DataMiner DataAggregator` service.
+
+1. Use the MigratorToGQIDxM tool with the following options:
+
+    | Option | Description |
+    |--|--|
+    | -u | Optional, use when another URL is configured in `appsettings.custom.json` (default is `http://localhost:12345`) |
+    | -p | Optional, use when DataAggregator is installed in custom location (default is `C:\Program Files\Skyline Communications\DataMiner DataAggregator`) |
+
+    Execute the following command with Administrator permissions:
+
+    ```bat
+    DataAggregator.MigratorToGQIDxM.exe
+    ```
+
+1. The output will confirm whether the migration succeeded.
+
+### Revert GQI DxM migration
+
+If you want to revert the GQI DxM migration, a backup of the query files is created.
+
+To revert:
+
+1. Stop the `DataMiner DataAggregator` service.
```

Comment:
Should the service also be stopped before running the migrator?

Reply:
No, the Data Aggregator service should be running or the migrator can't connect to the API